src/content/1.7/development/form-types/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Form Types Reference
+menuTitle: Form Types Reference
+weight: 20
+---
+
+## Form types reference
+
+While developing on PrestaShop you can use any form field type that is provided by Symfony framework.
+More about Symfony field types: https://symfony.com/doc/current/reference/forms/types.html.
```

Comment:
```suggestion
More about Symfony form types: https://symfony.com/doc/current/reference/forms/types.html.
```
It's called _form_ types in Symfony docs, so better to refer to the same name for consistency.